### PR TITLE
GH-15889 Fix thresholds_and_metric scores API for binomial models

### PR DESCRIPTION
**Pull request by maurever**
Issue: #15889

### CODE REVIEW

**Review by wendycwong — DISMISSED** [review]

File: h2o-py/h2o/model/metrics/binomial.py [hunk 1]
```diff
@@ -952,3 +952,27 @@ def gains_lift_plot(self, type="both", server=False, save_plot_path=None, plot=T
...
+        >>> h2o_data = h2o.H2OFrame(local_data)
+        >>> h2o_data.set_names(['response', 'predictor'])
+        >>> h2o_data["response"] = h2o_data["response"].asfactor()
+        >>> gbm = H2OGradientBoostingEstimator(ntrees=1,
+        ...                                    distribution="bernoulli")
+        >>> gbm.train(x=list(range(1,h2o_data.ncol)),
+        ...           y="response",
+        ...           training_frame=h2o_data)
+        >>> perf = gbm.model_performance()
+        >>> perf.
```
Comment (valenad1) [inline h2o-py/h2o/model/metrics/binomial.py:974]:
```suggestion
        >>> perf
```

File: h2o-r/tests/testdir_algos/gbm/runit_GBM_bernoulli.R [hunk 2]
```diff
@@ -44,17 +44,22 @@ test.GBM.bernoulli <- function() {
...
   print(RCM)
   Log.info("H2O Confusion Matrix:")
-  print(h2o.confusionMatrix(h2o.performance(prostate.h2o)))
+  perf <- h2o.performance(prostate.h2o)  
+  print(h2o.confusionMatrix(perf))
   
   R.auc <- gbm.roc.area(prostate.data$CAPSULE,R.preds)
   Log.info(paste("R AUC:", R.auc, "\tH2O AUC:", h2o.auc(h2o.performance(prostate.h2o))))
 
   # PUBDEV-515
```
Comment (valenad1) [inline h2o-r/tests/testdir_algos/gbm/runit_GBM_bernoulli.R:53]:
```suggestion
  #13556
```

**Review by valenad1 — CHANGES_REQUESTED** [review]
Couple of suggestion but looks good thank you!

I will approved when you make the changes because it will be dismissed.

File: h2o-r/tests/testdir_algos/gbm/runit_GBM_bernoulli.R (hunk 2 above)
Comment (maurever) [reply h2o-r/tests/testdir_algos/gbm/runit_GBM_bernoulli.R:53]: I don't think this change should be part of this PR. I added something new to this test, and this comment is completely unrelated.

Comment (valenad1) [reply h2o-r/tests/testdir_algos/gbm/runit_GBM_bernoulli.R:53]: All right, I'll do it in separete PR, its missed point from conversion to GH.

**Review by valenad1 — DISMISSED** [review]
👍

**Review by valenad1 — DISMISSED** [review]
👍

**Review by valenad1 — APPROVED** [review]
👍

**Pull request merged**